### ISSUE TITLE
buildx docs

### ISSUE THREAD

**Issue by chrisns** [docker-slim/docker-slim#152]
Hey, this looks amazing, is there anyway I can integrate with multi-arch docker builds?

Some docs on this would be amazing

Guessing it's something in the realm of `docker buildx create --driver [something]` and having a custom build driver? (sorry I don't know enough about the mechanics of it, hoping someone else does)

**Comment by kcq** [comment]
@chrisns just to double check... are you trying to use `docker-slim` to optimize an image for a different platform?

**Comment by chrisns** [comment]
Yeah. I guess a common use case is targeting a pi so building for arm but developing on amd64
Docker buildx makes that super easy, especially for easily collecting a multi arch manifest to push it all to a repo
Would be cool to be able to run docker-slim in that journey somehow

**Comment by kcq** [comment]
@chrisns It's possible in theory, but it'll have to rely only on static analysis because dynamic analysis requires the same platform. I'll need to explore the options there. This is an interesting idea

**Comment by chrisns** [comment]
in theory you could use the same qemu emulation pattern that buildx is to run the dynamic analysis - maybe ¯\_(ツ)_/¯

**Comment by openedhardware** [comment]
@kcq 

Any idea here?

We are using gitlab CI to build and deploy the docker images automatically.
Currently using buildx to build the arm images, but couldn't use docker-slim on the gitlab CI! 🙏
